docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Evolution of KLT
+description: Understand the paradigm of KLT, how it evolved from Keptn v1, and whether you should migrate to KLT
+weight: 5
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+The Keptn products provide tool-agnostic ways
+to manage and observe software deployment.
+Keptn connects to the tools you choose
+through an open event standard
+to automate delivery and remediation sequences.
+
+Two different products are available:
+
+* **First Generation:** Keptn v1.y.z LTS, referred to as "Keptn v1"
+* **Second Generation:** Keptn Lifecycle Toolkit, referred to as KLT
+
+In this page, we give a high-level overview
+of each of these products
+to help you understand the similarities and differences
+and which is most appropriate for your environment.
+
+> **Note**
+This section is under development.
+Information that is published here has been reviewed for technical accuracy
+but the format and content is still evolving.
+We hope you will contribute your experiences
+and questions that you have.
+
+## Keptn v1 (Keptn v1.y.z LTS)
+
+Development of Keptn v1 began in 2019.
+In 2022, it reached incubation status
+and the LTS (Long-Term Support) version was released.
+Keptn v1 is designed for the classic monolithic development world,
+where many developers work against a single Git repository
+on different components that have dependencies within that monolith.
+Software applications are developed
+using tools like Jenkins to run Continuous Integration.
+By building, testing, and validating the entire application
+each time a change is made,
+this creates a well-defined, monolithic application
+that operations could then deploy, observe, and operate.
+The complexity of the app was all in the monolithic repository.
+
+Keptn v1 is a general purpose orchestration engine
+for cloud and non-cloud native tooling on Kubernetes.
+It connects all different types of tools you use in your toolchain
+from build to test to deployment,
+including notification tools and monitoring tools,
+All these tools are connected
+through the CloudEvents open standard.
+
+This enables more automated delivery and remediation
+based on observability data.
+Observability is in the center of Keptn.
+Keptn v1 connects different testing and evaluation tools
+to execute sequences
+that ensure that deployments get into production safely.
+
+The quality gates feature enables you to define
+sophisticated analyses of the health of your deployment
+at each stage.
+They use Service-Level Indicators (SLIs),
```

Comment:
This sentence is confusing. Queries, that you define, are executed against your monitoring software (such as Prometheus, Dynatrace, or  Datadog). The metrics returned form Service-Level indicators (SLIs).